tools/ci_build/github/apple/test_ios_packages.py

```diff
@@ -144,6 +144,23 @@ def _test_ios_packages(args):
                 cwd=target_proj_path,
             )
 
+            if PackageVariant[args.variant] == PackageVariant.Full:
```

Comment:
Would it be easy enough to include macOS support in the Training package? Could be a follow up PR if it's non-trivial.